Cargo.toml

```diff
@@ -61,6 +64,9 @@ zonefile    = ["bytes", "std"]
 unstable-client-transport = []
 unstable-server-transport = ["chrono/clock", "hex", "libc", "tracing"]
 
+# Test features
+mock-time = ["mock_instant"]
```

Comment:
One issue with this is it causes you to always use "mock-time" if you build and/or test with `--all-features` :disappointed: